<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabScientists">
       <attribute name="title">
        <string>Scientists</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="filterScientist">
          <item>
           <widget class="QLineEdit" name="filterScientistsList">
            <property name="placeholderText">
             <string>Type to filter scientist list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ddmSortScientists">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listScientist"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="nameScientist">
          <item>
           <widget class="QLineEdit" name="addScientistName">
            <property name="placeholderText">
             <string>Name of scientist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="addScientistGender">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="yearScientist">
          <item>
           <widget class="QComboBox" name="ddmScientistBirth"/>
          </item>
          <item>
           <widget class="QComboBox" name="ddmScientistDeath"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="nationalityScientist"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="manageScientist">
          <item>
           <widget class="QPushButton" name="addScientist">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editScientist">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteScientist">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabComputers">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="filterComputer">
          <item>
           <widget class="QLineEdit" name="filterComputers">
            <property name="placeholderText">
             <string>Type to filter computer list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ddmComputerSort">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listComputer"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="nameComputer">
          <item>
           <widget class="QLineEdit" name="addComputerName">
            <property name="placeholderText">
             <string>Name of computer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ddmComputerBuilt">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="typeComputer">
          <item>
           <widget class="QComboBox" name="ddmComputerYear"/>
          </item>
          <item>
           <widget class="QLineEdit" name="addComputerType">
            <property name="placeholderText">
             <string>Type of computer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="manageComputer">
          <item>
           <widget class="QPushButton" name="addComputer">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editComputer">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteComputer">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTableLink">
       <attribute name="title">
        <string>Table Link</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="filterTableLink">
          <item>
           <widget class="QLineEdit" name="filterLinkedTables">
            <property name="placeholderText">
             <string>Type to filter linked table list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ddmTableLink">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listTableLink"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="tableLinkID">
          <item>
           <widget class="QLineEdit" name="addTableLinkSID">
            <property name="placeholderText">
             <string>Enter Scientist ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="addTableLinkCID">
            <property name="placeholderText">
             <string>Enter Computer ID</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="manageTableLink">
          <item>
           <widget class="QPushButton" name="addTableLink">
            <property name="text">
             <string>Add Link</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editTableLink">
            <property name="text">
             <string>Edit Link</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTableLink">
            <property name="text">
             <string>Delete Link</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
